network-store-client/src/main/java/com/powsybl/network/store/client/NetworkImpl.java

```diff
@@ -30,6 +33,44 @@ static NetworkImpl create(NetworkStoreClient storeClient, Resource<NetworkAttrib
         return new NetworkImpl(storeClient, resource);
     }
 
+    class BusBreakerViewImpl implements Network.BusBreakerView {
+
+        @Override
+        public Iterable<Bus> getBuses() {
+            return FluentIterable.from(getVoltageLevels())
```

Comment:
Avoid using guava FluentIterable. Just return a List created from java 8 streams and reuse it in getBusStream. See how it is done everywhere in this class.